tests/member_test.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+require_relative './test_helper'
+require_relative '../lib/member_page.rb'
+
+describe MemberPage do
+  around { |test| VCR.use_cassette('MemberPage', &test) }
```

Comment:
I strongly suspect that it would be better to name the cassette after the person whose page is being scraped, so that when we have more than one, it'll be obvious which is which.